gradle/init-scripts/overlapping-task-outputs-stats-init.gradle

```diff
@@ -34,7 +35,7 @@ gradle.addListener(new TaskExecutionAdapter() {
             type = type.superclass
         }
         def outputs = task.outputs.fileProperties
-            .findAll { it instanceof CacheableTaskOutputFilePropertySpec }
+            .findAll { it instanceof CacheableOutputFilePropertySpec }
```

Comment:
Should we deprecate and remove these init scripts at some point?

Reply:
We don't suggest to use those scripts any more so we can remove them right now as well. If anybody uses them, I would love to know about it.

Reply:
Let's do that as a separate PR.